src/test/java/org/folio/roles/it/RoleKeycloakIT.java

```diff
@@ -111,7 +112,7 @@ void getRole_negative_notFound() throws Exception {
   @Test
   @KeycloakRealms("classpath:json/keycloak/test-realm.json")
   void createRole_positive() throws Exception {
-    var roleToCreate = new Role().name("test role").description("test description");
+    var roleToCreate = new Role().name("test role").description("test description").type(RoleType.CONSORTIUM);
```

Comment:
can some assetion be added that newly created role has CONSORTIUM type?

Reply:
Good idea, added assersion for it